catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/managers/OrderMgrImpl.java

```diff
@@ -76,6 +78,10 @@ public void setQuerySources(List<String> querySources) {
         }
     }
 
+    public void setEmailSender(EmailSender emailSender) {
```

Comment:
I would add javadoc indicating that emailSender shouldn't be null and add a 'notNull' check in the code.